met-cron/src/met_cron/services/mail_service.py

```diff
@@ -53,17 +53,19 @@ def _send_email_notification_for_subscription(engagement_id, template_id, subjec
     def _render_email_template(engagement, participant, template):
         site_url = notification.get_tenant_site_url(engagement.tenant_id)
         tenant_name = EmailService._get_tenant_name(engagement.tenant_id)
-        metadata_model: EngagementMetadataModel = EngagementMetadataModel.find_by_id(engagement.id)
-        project_name = None
-        if metadata_model and 'project_name' in metadata_model.project_metadata:
-            project_name = metadata_model.project_metadata.get('project_name')
+        # TODO should be re-visited once the engagement metadata functionality of completed
```

Comment:
Thanks for the TODO comments. This is exactly what we want to see when we're commenting out code